holdings_maintenance/pds4/pds4archives.py

```
@@ -0,0 +1,554 @@
+
+    tarpath = pdsdir.archive_path_and_lskip()[0]
+    tar_tuples = read_archive_info(tarpath, logger=logger)
+
+    return validate_tuples(dir_tuples, tar_tuples, logger=logger)
+
+def repair(pdsdir, logger=None):
+
+    tarpath = pdsdir.archive_path_and_lskip()[0]
+    if not os.path.exists(tarpath):
+        logger = logger or pdslogger.PdsLogger.get_logger(LOGNAME)
+        logger.warn('Archive file does not exist; initializing', tarpath)
```

Ruff (0.14.0)

Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)